sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/AzureMLExecutePipelineActivity.cs

```diff
@@ -66,12 +77,15 @@ public AzureMLExecutePipelineActivity()
         /// will be passed in the continueOnStepFailure property of the
         /// published pipeline execution request. Type: boolean (or Expression
         /// with resultType boolean).</param>
-        public AzureMLExecutePipelineActivity(string name, object mlPipelineId, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), string description = default(string), IList<ActivityDependency> dependsOn = default(IList<ActivityDependency>), IList<UserProperty> userProperties = default(IList<UserProperty>), LinkedServiceReference linkedServiceName = default(LinkedServiceReference), ActivityPolicy policy = default(ActivityPolicy), object experimentName = default(object), object mlPipelineParameters = default(object), object mlParentRunId = default(object), object continueOnStepFailure = default(object))
+        public AzureMLExecutePipelineActivity(string name, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), string description = default(string), IList<ActivityDependency> dependsOn = default(IList<ActivityDependency>), IList<UserProperty> userProperties = default(IList<UserProperty>), LinkedServiceReference linkedServiceName = default(LinkedServiceReference), ActivityPolicy policy = default(ActivityPolicy), object mlPipelineId = default(object), object mlPipelineEndpointId = default(object), object version = default(object), object experimentName = default(object), object mlPipelineParameters = default(object), object dataPathAssignments = default(object), object mlParentRunId = default(object), object continueOnStepFailure = default(object))
```

Comment:
This is a breaking change. 
`miPipelineId` changed position. Since `mlPipelineEndpointId` and `version` are new parameters they should go at the end.

Reply:
For miPipelineId  what solution do you suggest as swagger did not think it is a break change when we move a parameter from required to optional.

For mlPipelineEndpointId and version, this will not a break change as they are not required and they have default value in old versions.

Reply:
This change seems to be approved.